packages/mongodb/_dev/deploy/variants.yml

```
@@ -0,0 +1,4 @@
+variants:
+  v4:
+    SERVICE_VERSION: 4
```

Oh well, or set the version here :slightly_smiling_face: 
```suggestion
  v4:
    SERVICE_VERSION: "4.4.8"
```

---

Got it